core/sector_storage/impl/remote_worker.hpp

```diff
@@ -84,7 +84,11 @@ namespace fc::sector_storage {
    private:
     explicit RemoteWorker(io_context &context);
 
+    IoThread worker_thread_;
+    IoThread httpSender_;
```

Comment:
Let's get context for send from out of class
```suggestion
```

Reply:
done